internal/tests/suite_memif_test.go

```diff
@@ -82,8 +82,7 @@ func newMemifVerifiableEndpoint(ctx context.Context,
 						up.NewServer(ctx, vppConn),
 						connectioncontext.NewServer(vppConn),
 						tag.NewServer(ctx, vppConn),
-						memif.NewServer(vppConn),
-						sendfd.NewServer(),
```

Comment:
Why is this removed?

Reply:
@denis-tingaikin `sendfd` server already present in this endpoint's chain(at the beginning)